x-pack/plugins/canvas/common/lib/__tests__/autocomplete.test.ts

```diff
@@ -4,63 +4,84 @@
-    expect(suggestions.length).to.be(functionSpecs.length);
-    expect(suggestions[0].start).to.be(0);
-    expect(suggestions[0].end).to.be(0);
+    expect(suggestions.length).toBe(functionSpecs.length);
+    expect(suggestions[0].start).toBe(0);
+    expect(suggestions[0].end).toBe(0);
   });
 
   it('should suggest functions filtered by text', () => {
     const expression = 'pl';
     const suggestions = getAutocompleteSuggestions(functionSpecs, expression, 0);
     const nonmatching = suggestions.map(s => s.text).filter(text => !text.includes(expression));
-    expect(nonmatching.length).to.be(0);
-    expect(suggestions[0].start).to.be(0);
-    expect(suggestions[0].end).to.be(expression.length);
+
+    expect(nonmatching.length).toBe(0);
+    expect(suggestions[0].start).toBe(0);
+    expect(suggestions[0].end).toBe(expression.length);
   });
 
   it('should suggest arguments', () => {
     const expression = 'plot ';
     const suggestions = getAutocompleteSuggestions(functionSpecs, expression, expression.length);
-    const plotFn = functionSpecs.find(spec => spec.name === 'plot');
-    expect(suggestions.length).to.be(Object.keys(plotFn.args).length);
-    expect(suggestions[0].start).to.be(expression.length);
-    expect(suggestions[0].end).to.be(expression.length);
+    const plotFn = getFunctionSpec('plot');
+
+    expect(suggestions.length).toBe(Object.keys(plotFn.args).length);
+    expect(suggestions[0].start).toBe(expression.length);
+    expect(suggestions[0].end).toBe(expression.length);
   });
 
   it('should suggest arguments filtered by text', () => {
     const expression = 'plot axis';
     const suggestions = getAutocompleteSuggestions(functionSpecs, expression, expression.length);
-    const plotFn = functionSpecs.find(spec => spec.name === 'plot');
+    const plotFn = getFunctionSpec('plot');
     const matchingArgs = Object.keys(plotFn.args).filter(key => key.includes('axis'));
-    expect(suggestions.length).to.be(matchingArgs.length);
-    expect(suggestions[0].start).to.be('plot '.length);
-    expect(suggestions[0].end).to.be('plot axis'.length);
+    expect(suggestions.length).toBe(matchingArgs.length);
+    expect(suggestions[0].start).toBe('plot '.length);
+    expect(suggestions[0].end).toBe('plot axis'.length);
   });
 
   it('should suggest values', () => {
+    const shape = 'shape';
     const expression = 'shape shape=';
     const suggestions = getAutocompleteSuggestions(functionSpecs, expression, expression.length);
-    const shapeFn = functionSpecs.find(spec => spec.name === 'shape');
-    expect(suggestions.length).to.be(shapeFn.args.shape.options.length);
-    expect(suggestions[0].start).to.be(expression.length);
-    expect(suggestions[0].end).to.be(expression.length);
+    const shapeFn = getFunctionSpec(shape);
+
+    expect(suggestions[0].start).toBe(expression.length);
+    expect(suggestions[0].end).toBe(expression.length);
+
+    if (shapeFn.name === shape && shapeFn.args.shape.options) {
```

Comment:
Checking `shapeFn.name` is `shape` and that `options` is not null gets them typed properly inside of the block. 

Should we add an else block to checks like this that just throws an error to ensure that all of the tests are being run?